src/testdir/test_cursorline.vim

```diff
@@ -192,7 +192,29 @@ func Test_cursorline_screenline()
     call term_sendkeys(buf, "gj")
     call TermWait(buf)
     call VerifyScreenDump(buf, 'Test_'. filename. '_18', {})
+    call term_sendkeys(buf, ":set breakindent& foldcolumn& signcolumn&\<cr>")
   endif
+  " showbreak should not be highlighted with CursorLine when 'number' is off
+  call term_sendkeys(buf, "gg0")
+  call term_sendkeys(buf, ":set list cursorlineopt=screenline listchars=space:-\<cr>")
+  call term_sendkeys(buf, ":set nonumber\<cr>")
+  call VerifyScreenDump(buf, 'Test_'. filename. '_19', {})
+  call term_sendkeys(buf, "fagj")
+  call TermWait(buf)
+  call VerifyScreenDump(buf, 'Test_'. filename. '_20', {})
+  call term_sendkeys(buf, "gj")
+  call TermWait(buf)
+  call VerifyScreenDump(buf, 'Test_'. filename. '_21', {})
+  call term_sendkeys(buf, "gj")
+  call TermWait(buf)
+  call VerifyScreenDump(buf, 'Test_'. filename. '_22', {})
+  call term_sendkeys(buf, "gj")
+  call TermWait(buf)
+  call VerifyScreenDump(buf, 'Test_'. filename. '_23', {})
+  call term_sendkeys(buf, "gj")
+  call TermWait(buf)
+  call VerifyScreenDump(buf, 'Test_'. filename. '_24', {})
+  call term_sendkeys(buf, ":set list& cursorlineopt& listchars&<cr>")
```

Comment:
> call term_sendkeys(buf, ":set list& cursorlineopt& listchars&<cr>")
excuse me, this was probably my fault. There is a backslash missing before the `<cr>` Sorry :(